clang/lib/Analysis/LifetimeSafety/Origins.cpp

```diff
@@ -157,4 +189,20 @@ const Origin &OriginManager::getOrigin(OriginID ID) const {
   return AllOrigins[ID.Value];
 }
 
+OriginID OriginManager::getOrCreate(const ValueDecl &D) {
```

Comment:
Is this a remnant for a wrong conflict resolution?

Reply:
Yes. Looks like a recent PR changed most of the logic in this file. Corrected the implementation.